### DISCUSSION TITLE
如何下载到程序所在服务器

### DISCUSSION THREAD

**Discussion by GaryRandom** [AlistGo/alist#3476] [Q&A]
我把ALIST挂载在NAS上，然后用另一台笔记本打开网页管理NAS。当我选择从ALIST挂载的云服务里下载文件时，文件经由NAS下载到笔记本。现在，我的需求是直接下载到NAS，不下载到笔记本，请问如何操作？

**Comment by xuegao456** [comment]
挂载一个NAS的本地盘，比如/local，然后选择复制到/local，后台会直接下载到/local
